ofl/modernantiqua/DESCRIPTION.en_us.html

```diff
@@ -1 +1,10 @@
-<p>This is another of my old fonts used for inscriptions on stone. I created the ModernAntiqua font around 1995 like my other fonts - when I started doing the inscriptions on stone professionally. The font is based on Roman square capitals. Initially the font contained only capitals and digits and later I made the missing lowercase and symbols. Last year I started converting my typeface designs into fonts. The first was NovaCut and all the Nova family, next MedievalSharp, and here's another...</p>
\ No newline at end of file
+<p>
+Modern Antiqua font, created by Wojciech Kalinowski around 1995 at the onset of him doing inscriptions on stone professionally, 
+is a font used for inscriptions on stone. The font is based on Roman square capitals. 
```

Comment:
Repetition of 'inscriptions on stone' :)

Also the 2nd sentence can move to its own line